build.gradle

```diff
@@ -20,7 +20,7 @@ publishing {
 }
 
 group = 'com.github.hmcts'
-version = '2.1.11'
+version = '2.1.12'
```

Comment:
This version now needs to be updated since we have released 2.1.12 today. Same applies to all other projects which uses this dependency :)